packages/components/src/JSONSchemaRenderer/JSONSchemaRenderer.test.js

```diff
@@ -245,6 +232,6 @@ describe('JSONSchemaRenderer', () => {
 			},
 		};
 		const wrapper = () => shallow(<JSONSchemaRenderer schema={schema} />);
-		expect(wrapper).toThrow(UnkownTypeException);
+		expect(wrapper).toThrow('Unkown type: unknown');
```

Comment:
Is it possible to fix `Unkown` (should be `Unknown`) in this PR? :)

Reply:
Done